pkgs/development/compilers/cakelisp/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchFromGitHub, clang, jam }:
+
+stdenv.mkDerivation {
+  pname = "cakelisp";
+  version = "unstable-2020-12-21";
+
+  src = fetchFromGitHub {
+    owner = "makuto";
+    repo = "cakelisp";
+    rev = "226bb4385a6fb6d61d37a5be8100004a140e5f65";
+    sha256 = "05fyj9a8m44kc3lzyqz4a8dnwa2rpcq981ryszsacywsng46nb01";
+  };
+
+  nativeBuildInputs = [ jam ];
+
+  buildInputs = [ clang ];
+
+  postPatch = ''
+    substituteInPlace src/Logging.hpp --replace 'log;' 'gLog;'
```

Comment:
Is there a patch upstream which fixes this?

Reply:
Not at the moment, it appears that this substitution is needed to fix the symbol conflict on darwin (where it conflicts with `math.h`'s `log`).

Reply:
Could you create a upstream PR and fetch that?

Reply:
This could be done in the future, repo is changing a lot currently

Reply:
Either do it now or never. I don't think anyone will remember this in 2 weeks.